app/views/groups/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= render "shared/nav" %>
+
+<div class="categories">
+  <% if @groups.empty? %>
+  <div class="get-started">
+    <h3 class="started">Get Started</h3>
+    <span class="div">Add your first budget by clicking on the button below</span>
+  </div>
+  <% else %>
+
+  <div class="categories-container">
+    <% @groups.each do |group| %>
+      <div class="category-card">
+        <div class="category-icon">
+          <%= image_tag group.icon, alt: 'icon', class: "icon" %>
+        </div>
+        <div class="card-text">
+        <%= link_to group_path(group.id.to_s), style: "text-decoration: none;" do %>
+          <p class="category-name"><%= group.name %></p>
+        <% end %>
+        <p>$<%= group.expenses.sum(:amount) %></p>
+        <%= link_to 'Learn more →', group_path(group.id.to_s), style: "text-decoration: none;", class: "learn" %>
+        <p class="user">by <%= current_user.name %></p>
+        </div>
+      </div>
+    <% end %>
+  <% end %>
+    <div class="text-center link-to">
+      <%= link_to 'Add New Category'.html_safe, new_group_path, class: "btn btn-success" %>
+    </div>
+  </div>
+</div>
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the `Add new category` button is placed at the bottom of the page as it is in the design guideline provided. this will enable a better UI.


Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="295" alt="Screenshot 2023-03-28 at 23 48 21" src="https://user-images.githubusercontent.com/58692590/228388678-72388486-a204-4c93-a84f-1e530fdbbc68.png"> | <img width="411" alt="Screenshot 2023-03-29 at 00 24 06" src="https://user-images.githubusercontent.com/58692590/228388777-daf0926b-c88d-47dc-b7c8-2483a5237efe.png">

Reply:
Thank you @Whoistolu , I will move it to the bottom.